WcaOnRails/app/controllers/registrations_controller.rb

```diff
@@ -58,6 +58,8 @@ def psych_sheet_event
   def index
     @competition = competition_from_params
     @registrations = @competition.registrations.accepted.includes(:user, :events).order("users.name")
+    @count_by_event = Hash.new(0)
+    @newcomers = @returners = 0
```

Comment:
I just feel like there's not reason for adding even more logic to the view, whereas the controller action is almost empty =)